### ISSUE TITLE
Add missing Focus rings

### ISSUE THREAD

**Issue by SabinaHrabinova** [bratislava/mestskakniznica.sk#308]
First step is to identify where focus ring are missing, probably needed to check in each section
Some focus rings are already implemented.
Plugin might be useful to try out.
Second step is implementation of those missing focus rings
